src/vstest.console/Internal/ConsoleLogger.cs

```diff
@@ -489,6 +508,7 @@ private void TestResultHandler(object sender, TestResultEventArgs e)
                 case TestOutcome.Skipped:
                     {
                         this.testsSkipped++;
```

Comment:
Do we need to have separate counts now that we are maintaining everything per dll ?

Reply:
removed seperate counts